integration/raft/channel_participation_test.go

```diff
@@ -261,9 +261,24 @@ var _ = Describe("ChannelParticipation", func() {
 
 			By("removing orderer1 from a channel")
 			channelparticipation.Remove(network, orderer1, "participation-trophy")
+			Eventually(func() channelparticipation.ChannelInfo {
+				return channelparticipation.ListOne(network, orderer1, "participation-trophy")
+			}, network.EventuallyTimeout).Should(Equal(channelparticipation.ChannelInfo{
+				Name:            "participation-trophy",
+				URL:             "/participation/v1/channels/participation-trophy",
+				Status:          "inactive",
+				ClusterRelation: "none",
+				Height:          0,
+			}))
```

Comment:
Is this because the channel is still pending removal? If so this may introduce a race and a flakey test, because if the test thread is slower then the orderer1 process, you won't get to see this transitory state.

Reply:
Yes agreed, see previous comment